core/src/main/java/org/apache/carbondata/core/util/DataTypeUtil.java

```diff
@@ -436,10 +436,11 @@ public static boolean isFixedSizeDataType(DataType dataType) {
    *
    * @param dataInBytes    data
    * @param actualDataType actual data type
+   * @param isTimeStampConversion
    * @return actual data after conversion
    */
   public static Object getDataBasedOnDataTypeForNoDictionaryColumn(byte[] dataInBytes,
-      DataType actualDataType) {
+      DataType actualDataType, boolean isTimeStampConversion) {
```

Comment:
Add one more method to pass the `isTimeStampConversion`

Reply:
Done